scribble-doc/scriblib/scribblings/autobib.scrbl

```diff
@@ -90,6 +92,29 @@ section for the bibliography. It has the contract
 (->* () (#:tag string? #:sec-title string?) part?)
 ]
 
+If provided, the function bound to @racket[cite-author]
```

Comment:
Instead of putting "If provided," everywhere, I think it would be clearer to edit the first paragraph to say:

> Binds ~cite-id, citet-id, generate-bibliography-id, (optionally) `cite-author-id`, and (optionally) `cite-year-id`, which share state to accumulate and render citations.